docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -410,6 +410,54 @@
                     </td>
                 </tr>
                 <tr>
+                                              <!-- change link to StandAlone design/tutorial doc -->
+                <th colspan="3" class="section" id="StandAlone"><a href="../index.html">StandAlone job configuration</a></th>
```

Comment:
Can we just call it "Zk based Job Coordination" ? Same for html element ids as well?